packages/mdc-auto-init/README.md

```diff
@@ -160,3 +160,10 @@ warning, you could simply pass in a nop.
 ```
 
 This will suppress any warnings about already initialized elements.
+
+### Events
+
+#### MDCAutoInit:End
+Triggered when initialization of all components is complete.
```

Comment:
If that's the case, we probably shouldn't rely on component to emit event.

See my comment below